Formula/pilosa.rb

```diff
@@ -0,0 +1,63 @@
+class Pilosa < Formula
+  desc "Distributed bitmap index that queries across data sets"
+  homepage "https://www.pilosa.com"
+  url "https://github.com/pilosa/pilosa/archive/v0.3.1.tar.gz"
```

Comment:
@rakkesh I'm going to bump it because it's good to stay current - will update the title and everything else after I'm done.